test/axl_cp.c

```diff
@@ -0,0 +1,195 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <string.h>
+#include <limits.h>
+#include <errno.h>
+#include <sys/stat.h>
+#include <signal.h>
+#include "axl.h"
```

Comment:
Looks like the errors are happening when building `axl_cp.c`. Do you need to include `pthread.h` or `axl_pthread.h` here?